### ISSUE TITLE
Central storage for preset data (names, skins, etc) like pluralkit

### ISSUE THREAD

**Issue by Conquerix** [sisby-folk/switchy#63]
Hi,

I'm currently making several fabric servers behind a proxy and wondered if it was possible to synchronize all the presets, inventories, etc. across all the servers.

Is there a built-in way of doing so?
or maybe by simply synchronizing some player data and other files?

**Comment by sisby-folk** [comment]
Yes, you can do this using any existing utility that can copy player data - switchy doesn't store any of it's data outside of the player data file. If you find a specific utility that works, feel free to post it here for others to find.

**Comment by Conquerix** [comment]
Thank you for answering, I'll test what I can find, for now I'm gonna try to synchronise the player.dat files with Syncthing, adding some delay in the server transfer should do the job.
Also HuskSync is being ported to fabric, once it is done I'll also test its compatibility with Switchy.

**Comment by Conquerix** [comment]
Hey just a heads up, symlinking the player data accross servers works perfectly. (Of course even the location of the players of synced so it's far from perfect but it works)
